operations/operations/publishing/publishing-guidelines/brand-and-visual-design-guidelines.md

```diff
@@ -113,6 +113,6 @@ Alternatively, community projects may concatenate names containing Mattermost, f
 
 ## Company Short Description
 
-Mattermost is an open source Slack-alternative built for enterprise. Thousands of companies use Mattermost for secure, compliant workplace messaging across web, PC and phones with archiving, search, corporate SSO, and integration with over 700 third-party applications. Available under MIT license in 16 languages, Mattermost offers peace-of-mind, value, control, and freedom from lock-in for organizations around the world.
+Mattermost is an open source Slack-alternative built for enterprise. Thousands of companies use Mattermost for secure, compliant workplace messaging across web, PC, and phones; leveraging archiving, search, corporate SSO, and integration with hundreds of third-party applications. Available under MIT license in 16 languages, Mattermost offers peace of mind, value, control, and freedom from lock-in for organizations around the world. Learn more at https://mattermost.com/.
```

Comment:
Same note as in https://github.com/mattermost/docs/pull/3687:

Given this is a text that we recommend people to use as a short description for work they've done, e.g. an integration, wondering if "leveraging" is easily understood by non-English audience?

Otherwise the change looks good to me :)

Reply:
Great point. I think reverting to "with" is best. No need to over-complicate it.